ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/execution/PullQueryExecutorMetrics.java

```diff
@@ -78,12 +77,10 @@ public void recordRemoteRequests(final double value) {
     this.remoteRequestsSensor.record(value);
   }
 
-  public void recordRate(final double value) {
-    this.requestRateSensor.record(value);
-  }
-
-  public void recordLatency(final double value) {
-    this.latencySensor.record(value);
+  public void recordLatency(final long nanoSeconds) {
```

Comment:
As suggested above, change to taking the end time, rather than the elapsed time?